docs/lifespan.rst

```diff
@@ -0,0 +1,76 @@
+Lifespan
+==============================
+The Lifespan QoS establish the maximum validity period of the samples saved on the entity history. Each of those
+samples has a lifespan timer associated so that once it expires the linked data is removed from the entity history.
```

Comment:
Again, maybe too technical (also not strictly speaking correct, as there is one timer shared among samples, not one for each sample). You could just say something like:

> The lifespan QoS establishes the maximum validity period of the samples saved on an entity's history. When the lifespan period elapses, the corresponding sample is automatically removed.

If you want, you could also add to the previous sentence:

> Unlike the Deadline Qos, there is no callback to inform the user when this happens

(the latter is optional, just add it if you want).